contracts/starknet/authenticators/eth_tx.cairo

```diff
@@ -43,7 +43,14 @@ end
 
 @external
 func execute{syscall_ptr : felt*, range_check_ptr, pedersen_ptr : HashBuiltin*}(
-    target : felt, function_selector : felt, calldata_len : felt, calldata : felt*
+    target : felt,
+    function_selector : felt,
+    calldata_len : felt,
+    calldata : felt*,
+    signer_len : felt,
+    signer : felt*,
+    signature_len : felt,
```

Comment:
Any reason to have signature for eth tx?

Reply:
The interface has been modified so every authenticator should have this now (can be set to 0 if the auth doesn't need it)

Reply:
Why would you want same interface for authenticators?

Reply:
I don't know, there currently is an `i_authenticator.cairo` so I thought we had good reasons to want an interface. I guess if we wish to make sure our contracts are all easily integratable then it makes sense to have an authenticator interface, but it's true we can do without it.

Reply:
I imagine these authenticators more external to SX, others apps may want to use these auths I'd prefer each auth to be custom tailored for their use case and not depending on the parameters of others auths